src/components/TasksTable/index.jsx

```diff
@@ -2,26 +2,47 @@ import { Component, Fragment } from 'react';
 import TableCell from '@material-ui/core/TableCell';
 import TableRow from '@material-ui/core/TableRow';
 import Chip from '@material-ui/core/Chip';
+import Drawer from '@material-ui/core/Drawer';
+import IconButton from '@material-ui/core/IconButton';
 import List from '@material-ui/core/List';
 import ListItem from '@material-ui/core/ListItem';
 import ListItemText from '@material-ui/core/ListItemText';
 import Button from '@material-ui/core/Button';
 import { withStyles } from '@material-ui/core/styles';
 import LinkIcon from 'mdi-react/LinkIcon';
+import InformationVariantIcon from 'mdi-react/InformationIcon';
```

Comment:
We are still important the old icon. Change to `mdi-react/InformationVariantIcon`.

Reply:
My bad.. so thats why i feel like there is no difference